modules/jaeger-upgrading-es.adoc

```diff
@@ -0,0 +1,12 @@
+////
+This PROCEDURE module included in the following assemblies:
+- jaeger_install/rhbjaeger-updating
+- service_mesh_install/updating-ossm
+////
+
+[id="upgrading_es5_es6_{context}"]
+= Upgrading from Elasticsearch 5 to Elasticsearch 6
+
+There is an issue with certificates when upgrading from Elasticsearch 5 to Elasticsearch 6.
+
+As a workaround, you must delete your Jaeger instance, then recreate the Jaeger instance.  Re-creating the Jaeger instance triggers creating a new set of certificates.
```

Comment:
We should document that if persistent storage is used the new instance will continue using it - assuming the Jaeger instance name and namespace didn't change.

Reply:
Although the same should be documented on https://docs.openshift.com/container-platform/4.4/jaeger/jaeger_install/rhbjaeger-deploying.html. 

The upstream docs contain this information in https://www.jaegertracing.io/docs/1.18/operator/#self-provisioned see `<2>`